azure-function/yield-curve/__init__.py

```diff
@@ -24,9 +23,13 @@ def main(req: func.HttpRequest, rates) -> func.HttpResponse:
     """
     logging.info('Python HTTP trigger function processed the rates API request.')
 
-    return func.HttpResponse(rates, mimetype="application/json")
-
+    param = get_param(req=req, param="data_format")
 
+    if param == "csv":
+        rates_df = pd.read_json(rates)
+        return func.HttpResponse(rates_df.to_csv(), mimetype="text/csv")
+    else:
+        return func.HttpResponse(rates, mimetype="application/json")

```

Comment:
Have you tested the API now without passing date_format? Does it work returning the json?

Reply:
I'm asking because I'm wondering if the param == "csv" will work if the previous line returns nothing. In case it doesn't an additional if None check might be required (e.g. `if (param is not None) and (param == "csv")`). But if it does work like this, perfect :)

Reply:
I tested without data_format and it will return a json.

Reply:
![image](https://user-images.githubusercontent.com/45222861/88296991-abd7e100-ccff-11ea-8893-81f90260eed9.png)